### PR TITLE
BaSD-MS-56: Final form counselors

### PR DESCRIPTION
**Pull request by MicaQuarleri**

### CODE REVIEW

**Review by aquilesmariani — CHANGES_REQUESTED** [review]
Almost done!

I'm facing this when I get into the counselor form, we should fix it
Screen Shot 2021-12-22 at 16 41 24

**Review by ignacioaristo — APPROVED** [review]
Looks good to me

File: package-lock.json [hunk 1]
```diff
@@ -9200,6 +9200,10 @@
       "version": "4.20.4",
       "resolved": "https://registry.npmjs.org/final-form/-/final-form-4.20.4.tgz",
       "integrity": "sha512-hyoOVVilPLpkTvgi+FSJkFZrh0Yhy4BhE6lk/NiBwrF4aRV8/ykKEyXYvQH/pfUbRkOosvpESYouFb+FscsLrw==",
+<<<<<<< HEAD
+=======
+      "peer": true,
+>>>>>>> 76483c2333de1647a53b3655fba1d287edb70e68
```
Comment (aquilesmariani) [inline package-lock.json:9206]: You need to fix this instead of committing the conflicts

File: package-lock.json [hunk 2]
```diff
@@ -29130,6 +29134,10 @@
...
```
Comment (aquilesmariani) [inline package-lock.json:29140]: This one too

**Review by aquilesmariani — CHANGES_REQUESTED** [review]
Check that you didn't resolve conflicts in your last merge

**Review by leimeterjoaquin — CHANGES_REQUESTED** [review]

File: src/Components/Shared/Input/index.js [hunk 3]
```diff
@@ -1,33 +1,18 @@
 import styles from './input.module.css';
```
[the remark this answers is not in the document]
Comment (leimeterjoaquin) [reply src/Components/Shared/Input/index.js:1]: profiles form doesn't work, for example

**Review by aquilesmariani — APPROVED** [review]
Looks great!

**Pull request merged**